### DISCUSSION TITLE
No Internet connection on Win10 WSL2 until I start the WSL2 Ubuntu 22

### DISCUSSION THREAD

**Discussion by TheMasterFX** [podman-desktop/podman-desktop#3402] [Q&A]
A couple of weeks ago I installed podman-desktop. At first I didn't get it to work since I did not get an internet connection to even download a container image.
After reconfiguring all my ethernet settings which I found on Stackoverflow etc. which also didn't help, I finally find out that when I start the Ubuntu which comes with the WSL2 the connection in a podman container magically works.

Now when I want to use podman, I first have to start ubuntu so that the internet connection in the container works. But I find that a little bit annoying.

**Comment by benoitf** [comment]
cc @n1hility @gbraad
